src/modules/endpoint/DatabaseConnector.js

```diff
@@ -146,6 +156,9 @@ class DatabaseConnector {
     return `${CACHE_DIR_PATH}/cities.json`
   }
 
+  /**
+   * Prior to storing lastUpdate, there needs to be a lastUsage of the city in the context.
```

Comment:
```suggestion
   * Prior to storing lastUpdate, the lastUsage of the city has to be set.
```
Context is misleading here imo.